priv/stats_descriptions.cfg

```diff
@@ -51,8 +51,8 @@
     {desc, <<"number of _local document write operations">>}
 ]}.
 {[couchdb, httpd, bulk_requests], [
-    {type, counter},
-    {desc, <<"number of bulk requests">>}
+    {type, histogram},
+    {desc, <<"distribution of bulk request sizes">>}
```

Comment:
Yeah, I'm stuck on names. The 'n' value in here is the same value as the original counter, which is why I felt the name was appropriate, it just has more information, like the distribution of the sizes of these requests. That you can have multiple things inside a _bulk_docs is what distinguishes it from most other endpoints, so I felt that implied what the histogram meant (and the description makes it explicit)